raft-ann-bench/cpu/Dockerfile

```
@@ -3,7 +3,7 @@
 ARG PYTHON_VER=3.11
 ARG RAPIDS_VER=24.04
 
-FROM condaforge/mambaforge:23.3.1-0 as raft-ann-bench-cpu
+FROM condaforge/mambaforge:23.11.0-0 as raft-ann-bench-cpu
```

Should this be updated to miniforge3?
```suggestion
FROM condaforge/miniforge3:23.11.0-0 as raft-ann-bench-cpu
```